scripts/log-analysis/false_transit.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
```

Comment:
Can we update this script for how the bot is detecting false transits: 

// To determine no forward progress:
//    If the vehicle is in the vertical position; pitch > resolve_pitch_threshold  (default 30 deg)
//    If the vehicle desired speed is > resolve_desired_speed_threshold (default: 0 m/s)
//    How long to give the vehicle to get to the horizontal position: resolve_no_forward_progress_timeout (default: 15 sec)